svelte/components/Nft/NftsDisplayMode.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+  export let mainContentDisplayComponent: string = "grid";
```

Comment:
use simplier name like "mode", as component, will not be always related to main content

Reply:
done